idaes/ui/fsvis/static/js/cell_config.js

```diff
@@ -61,9 +66,27 @@ export class JointJsCellConfig {
                 y: b.y + gap.destination.y
             };
 
+            // Getting the current angles of the icons to calculate the right
+            // position for the original points calculated above
+            const angle_a = linkView.getEndView('source').model.angle();
+            const angle_b = linkView.getEndView('target').model.angle();
+
+            var vector_a = new g.Point(p1.x, p1.y);
```

Comment:
why are you calling them `vector_a` and `vector_b`? They seem to be points still, not direction and magnitude from the origin (despite being rotated around some origin). Also using 'a' and 'b' hides some semantics. How about calling it 'src' and 'tgt' and simply using those instead of origin_a and origin_b?
```
src_rotated = p1.rotate(src, src_orth * src_angle);
dst_rotated = p2.rotate(dst, dst_orth * dst_angle);
return joint.routers.manhattan([src_rotated, ...vertices, dst_rotated], opt, linkView);
```